alembic_migration/versions/24f8da659c78_add_ratings_to_outings.py

```diff
@@ -0,0 +1,153 @@
+"""Add ratings to outings
+
+Revision ID: 24f8da659c78
+Revises: 8c230a4a0284
+Create Date: 2017-10-13 14:05:52.606170
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '24f8da659c78'
+down_revision = '8c230a4a0284'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        'outings',
+        sa.Column('height_diff_difficulties', sa.SmallInteger(),
+                  nullable=True),
+        schema='guidebook')
+
+    op.add_column(
+        'outings',
+        sa.Column('ski_rating',
+                  sa.Enum(
+                      '1.1', '1.2', '1.3', '2.1', '2.2', '2.3', '3.1', '3.2',
+                      '3.3', '4.1', '4.2', '4.3', '5.1', '5.2', '5.3', '5.4',
+                      '5.5', '5.6', name='ski_rating', schema='guidebook'),
+                  nullable=True),
+        schema='guidebook')
+
+    op.add_column(
+        'outings',
+        sa.Column('labande_ski_rating',
+                  sa.Enum(
+                      'S1', 'S2', 'S3', 'S4', 'S5', 'S6', 'S7',
+                      name='labande_ski_rating', schema='guidebook'),
+                  nullable=True),
+        schema='guidebook')
+
+    op.add_column(
+        'outings',
+        sa.Column('global_rating',
+                  sa.Enum(
+                      'F', 'F+', 'PD-', 'PD', 'PD+', 'AD-', 'AD', 'AD+', 'D-',
+                      'D', 'D+', 'TD-', 'TD', 'TD+', 'ED-', 'ED', 'ED+', 'ED4',
+                      'ED5', 'ED6', 'ED7', name='global_rating',
+                      schema='guidebook'),
+                  nullable=True),
+        schema='guidebook')
+
+    op.add_column(
+        'outings',
+        sa.Column('engagement_rating',
+                  sa.Enum('I', 'II', 'III', 'IV', 'V', 'VI',
+                          name='engagement_rating',
+                          schema='guidebook'),
+                  nullable=True),
+        schema='guidebook')
+
+    op.add_column(
+        'outings',
+        sa.Column('equipment_ratings',
```

Comment:
``equipment_ratings`` => ``equipment_rating`` (no trailing S)
I guess that's the/a reason why Travis fails?